provider/oci/images.go

```diff
@@ -381,37 +414,6 @@ func instTypeByShapeName(shape string) string {
 	}
 }
 
-var (
-	oracleAmdBm    = fmt.Sprintf("%s|%s|%s|%s", string(ociCore.ShapePlatformConfigOptionsTypeAmdMilanBm), string(ociCore.ShapePlatformConfigOptionsTypeAmdRomeBm), string(ociCore.ShapePlatformConfigOptionsTypeIntelSkylakeBm), string(ociCore.ShapePlatformConfigOptionsTypeIntelIcelakeBm))
-	oracleAmdBmGpu = fmt.Sprintf("%s|%s", string(ociCore.ShapePlatformConfigOptionsTypeAmdMilanBmGpu), string(ociCore.ShapePlatformConfigOptionsTypeAmdRomeBmGpu))
-	oracleAmd      = fmt.Sprintf("%s|%s", string(ociCore.ShapePlatformConfigOptionsTypeAmdVm), string(ociCore.ShapePlatformConfigOptionsTypeIntelVm))
-)
-
-// archREs maps regular expressions for matching
-// oracle architectures and instance types to juju
-// values
-var archREs = []struct {
-	*regexp.Regexp
-	arch         string
-	instanceType string
-}{
-	{regexp.MustCompile(oracleAmd), arch.AMD64, VirtualMachine.String()},
-	{regexp.MustCompile(oracleAmdBm), arch.AMD64, BareMetal.String()},
-	{regexp.MustCompile(oracleAmdBmGpu), arch.AMD64, GPUMachine.String()},
-}
-
-// normaliseArchAndInstType returns the Juju architecture and instance type
-// corresponding to a shape's reported architecture and instance type based
-// off ShapePlatformConfigOptionsTypeEnum.
-func normaliseArchAndInstType(val ociCore.ShapePlatformConfigOptionsTypeEnum) (string, string) {
```

Comment:
You still need to handle the InstanceTypes here, VM, BM etc for use.

Reply:
this is done in `instTypeByShapeName()` which I didn't touch